### PR TITLE
[2.11.x] DDF-3229 Updated documentation for some Intrigue config

### PR DESCRIPTION
**Pull request by rzwiefel**
#### What does this PR do?
Updates documentation for a configuration option in Intrigue
#### Who is reviewing it? 
@mcalcote @vinamartin 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
@codice/docs 
#### Choose 2 committers to review/merge the PR. 
@clockard
@ricklarsen - Documentation

#### How should this be tested? (List steps with links to updated documentation)
build
#### What are the relevant tickets?
[DDF-3229](https://codice.atlassian.net/browse/DDF-3229)
#### Checklist:
- [x] Documentation Updated
- [n/a] Update / Add Unit Tests
- [n/a] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bdeining** [comment]
build now

**Comment by clockard** [comment]
build now

File: distribution/docs/src/main/resources/content/_tables/org.codice.ddf.catalog.ui.config-table-contents.adoc [hunk 1]
```diff
@@ -235,5 +235,12 @@ TMS example (3d map support only): `[{"name": "Example TMS", "show": true, type"
 |
 |true
 
+|Maximum Endpoint Upload Size
+|maximumUploadSize
+|Integer
+|The maximum size of data that the Intrigue REST endpoints will receive. Please note this is not related to the product upload size, this only refers to the json data sent and received from the Intrigue endpoints.
+|1048576
+|false
+
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_tables/org.codice.ddf.catalog.ui.config-table-contents.adoc]: This file was recently renamed in another PR.

**Comment by rzwiefel** [comment]
build now

**Comment by clockard** [comment]
build now

**Pull request merged**